src/index.ts

```diff
@@ -20,8 +24,14 @@ const run = async () => {
     jsonSummary, jsonFinal 
   });
 
+  // Indicate working directory (if set) in Action summary for easier understanding (ie. in a monorepo)
```

Comment:
... same for this comment here - I think the code is still simple enough to understand what is going on 🙂

Reply:
Yeah, this was a hold over some previous work where I didn't have separate variables but rather more string concatenation (was harder to follow), and I never removed it after refactoring; thanks!